packages/expo-firebase/src/app/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "expo-firebase/app",
```

Comment:
This is not a valid package name AFAIK.. delete this file?

Reply:
This is an alias for accessing the contents of that folder.

Reply:
How does this work, though? If you require `"x/y"`, the Node resolution algorithm is to look in `node_modules/x/y`, rather than finding all the package.json files first and looking at the "name" fields to define a module with that name. This seems like a potential bug in Metro if it supports slashes in package names.